eva-lib/src/main/java/uk/ac/ebi/eva/lib/models/VariantStudy.java

```diff
@@ -116,14 +115,15 @@ public VariantStudy(String studyName, String studyId) {
     }
 
     public VariantStudy(String studyName, String studyId, List<VariantSource> sources) {
-        this(studyName, studyId, sources, null, new int[0], null, null, null, null, null, null, StudyType.COLLECTION, 
-             null, null, null, null, null, new String[0], -1, -1);
+        this(studyName, studyId, sources, null, new int[0], null, null, null, null, null, null, StudyType.COLLECTION,
+             null, null, null, null, null, new String[0], -1);
     }
 
-    public VariantStudy(String studyName, String studyId, List<VariantSource> sources, String description, int[] taxonomyId, 
-            String speciesCommonName, String speciesScientificName, String sourceType, String center, String material, 
-            String scope, StudyType type, String experimentType, String experimentTypeAbbreviation, String referenceAssembly, 
-            String platform, URI projectUrl, String[] publications, int numVariants, int numSamples) {
+    public VariantStudy(String studyName, String studyId, List<VariantSource> sources, String description,
+                        int[] taxonomyId, String speciesCommonName, String speciesScientificName, String sourceType,
+                        String center, String material, String scope, StudyType type, String experimentType,
+                        String experimentTypeAbbreviation, String referenceAssembly, String platform, URI projectUrl,
+                        String[] publications, int numSamples) {
```

Comment:
Didn't we find that DGVa supports multiple species and assemblies in a single study? Or is that refactoring intended for a future PR? I am fine with splitting it in 2 if that facilitates merging.